### PR TITLE
B.Protocol

### PR DESCRIPTION
**Pull request by yaronvel**
#### Hi! What is your PR about?

- [x] Add images for your space
- [ ] Add custom domain for your space
- [x] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/aliases.json [hunk 1]
```diff
@@ -19,5 +19,7 @@
   "cover": "cover-protocol.eth",
   "iwan": "iwan.eth",
   "oceandao": "officialoceandao.eth",
-  "macaronswap": "macaronswap.eth"
+  "macaronswap": "macaronswap.eth",
+  "mBProtocol" : "m.bsnapshot.eth",
+  "cBProtocol" : "bsnapshot.eth"
```
Comment (gokaiorg) [inline spaces/aliases.json:24]: When creating a space you do not have to edit the `aliases.json` file.
```suggestion
  "macaronswap": "macaronswap.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/aliases.json (hunk 1 above)
Comment (yaronvel) [reply spaces/aliases.json:24]: But we want to add two aliases, one for mbprotocol and one for cbprotocol

Comment (gokaiorg) [reply spaces/aliases.json:24]: You have to create an alias if you had a space that was created before the implementation of ENS and I can't find https://snapshot.org/#/mbprotocol and https://snapshot.org/#/cbprotocol. You are rather on a space creation and you don't need an alias

Comment (gokaiorg) [reply spaces/aliases.json:24]: Maybe you want to make a subdomain? https://docs.snapshot.org/spaces/add-custom-domain

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Images looks good to me!

File: spaces/aliases.json (hunk 1 above)
Comment (yaronvel) [reply spaces/aliases.json:24]: oh, i think i misunderstood the feature.
We want it to be hosted on snapshot.org, however we don't want it to include `.eth` and potentially other characters as well.
E.g., like `https://snapshot.org/#/dai` doesn't have `.eth`.
Are you saying this is no longer possible?
If it isn't, then we will use our own custom domain.

Comment (gokaiorg) [reply spaces/aliases.json]: > Are you saying this is no longer possible?
Yes, now the spaces are created with ENS and have the domain extension .eth or .xyz. The solution is to create your own subdomain.

Comment (yaronvel) [reply spaces/aliases.json]: I submitted a fix that will only use a domain.
Sorry for the confusion.

**Comment by gokaiorg** [comment]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by yaronvel** [comment]
> There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

done

**Comment by gokaiorg** [comment]
@yaronvel There is still conflict in your RP. You need to update your local before you start committing again.

**Comment by yaronvel** [comment]
> @yaronvel There is still conflict in your RP. You need to update your local before you start committing again.

i keep resolving conflicts but then new projects are being merged before my PR.
Updated again.
Please merge my PR before others :)

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**